packages/eslint-plugin-package-json/package.json

```diff
@@ -18,10 +18,10 @@
 	],
 	"license": "GPL-2.0-or-later",
 	"dependencies": {
-		"@wordpress/npm-package-json-lint-config": "^4.24.0",
+		"@wordpress/npm-package-json-lint-config": "^4.25.0",
 		"eslint-plugin-json-es": "^1.5.7",
 		"esquery": "^1.4.0",
-		"npm-package-json-lint": "^5.4.2"
+		"npm-package-json-lint": "^6.0.0"
```

Comment:
We needed to bump this one as it was bumped in `@wordpress`. I didn't spot anything breaking that would affect us ([release notes](https://github.com/tclindner/npm-package-json-lint/releases/tag/v6.0.0)).